### DISCUSSION TITLE
HOw do we download XD library file?

### DISCUSSION THREAD

**Discussion by skybluemizu** [dxc-technology/halstack#3] [General]
Hi! how do we doenload this component library? I'm from UX team UK, and it would be extremely helpful for our new project.
Have you user tested form components?
TA!

**Comment by aweell** [comment]
Hi @skybluemizu,

UI-Kit can be downloaded from [here](https://github.com/dxc-technology/halstack-style-guide). Our form components are in our roadmap for design review, until now they have not been user tested.

Any [feedback](https://github.com/dxc-technology/halstack-style-guide/issues) related to `data-input` components or any other component present in Halstack Design System is more than welcomed!

**Comment by skybluemizu** [comment]
You're a star!!!

Re design review; would you like some help? I've spent last 4 years working with banks, UK government and accessibility aganecy to producte best practice documentation. We have also (as a bank :P) tested a lot of forms and other components with various user types. hahaha not trying to brag :P but we have started to collect that stuff here (UK) before I've found out that you exist :) 
For example forms in DXC official styleguide wouldn't pass usability or accessibility standards for gov - and they actually started to be the leaders in that feild
Let me know if we can join forces, or if you think I'm being a total prick, just ignore this message :)

**Comment by aweell** [comment]
Hey @skybluemizu,

Let's connect! 👍   

Idk how to reach since there is no DM in GitHub.

You can find me through Teams / Outlook: **Alejandro Rodríguez Bueno**

**Comment by skybluemizu** [comment]
Hey Alejandro - MS is being stupid and can't find you, can you send me a message :) mrudecka@dxc.com
